docs/sources/mimir/_index.md

```diff
@@ -1,29 +1,64 @@
+
```

Comment:
I believe doc-validator is confused because the front matter starts on line 2 instead of the first line in the file.

```suggestion
```